plugins/continuity.js

```diff
@@ -1382,6 +1382,13 @@
 					src = elements[i].currentSrc ||
 						elements[i].src ||
 						(typeof elements[i].getAttribute === "function" && elements[i].getAttribute("xlink:href"));
+	
+					// if src if not defined, look for it in css background image
+					if (!src) {
+						var bgStyle = BOOMR.window.getComputedStyle(elements[i]).getPropertyValue('background');
+						var bgImgUrl = bgStyle.match(/url\(["']?([^"']*)["']?\)/)[1];
```

Comment:
This will throw an error if `match` returns `null`.

Reply:
Oups! forgot about the array while testing, just pushed a fix.